### ISSUE TITLE
Multiple findings...

### ISSUE THREAD

**Issue by Bugmaniac71** [comixed/comixed#1437]
**BEFORE OPENING A TICKET, please consider posting your feature request to the user mailing list.* <<-- Subscribed to mailing list but nothing showing up in it so dont know if this has already been raised - sorry if it has.

1. Ability to select comics anywhere NOT IN COMIC VIEW. Annoying that have to use cover view to select anything.
2. More robust UNRAR. Craps out  at slightest issue and hangs.
3. Faster import - imported 60 comics and it took 20 minutes - why so slow?
4. Scrape system - Undo feature? Ability to edit data on one page rather than multiple tabs.
5. Selection system - If I select show only CBR, then want to select all those CBR's ONLY it selects every format again. 

Other than that it has potential. How do you utilitse a different database system?

Again apologies if these are in the mailing list but I cant access that. (Why a mailing list may i add, a Wiki and FAQ are easily more flexible....)
Regards
B

**Comment by mcpierce** [comment]
@Bugmaniac71 Thank you for the feedback. I'm very open to feedback and ideas, and maybe you can start a conversation in the mailing list with others so we can kick these area and come up with some ideas?

1. What sort of selection flow do you see?
1. We are using a third-party library for performing unrar. I can check to see if there's an updated release of their library.
1. The import does a *lot* of work when first processing comics, such as extracting the individual pages and getting a checksum on them to look for blocked and duplicated pages, as well as building up the database with file details. We're always looking at ways to improve performance, though.
1. Undoing scraping would be an additional feature, sure. The expectation is that you've selected the comic that's the correct one. As for editing on multiple tabs, I'm not sure I follow that; i.e., you can only edit a comic on the **Edit Details** tab.
1. This is a bug. Can you please open a bug report? I'll get a fix out for that straight away.

Regarding the mailing list: wikis and FAQs are read only, while mailing lists let people contribute to discussions. You say you can't access that, what does that mean? If you mean you can't send to it: you can only post to a mailing list if you're subscribed to it. You can, though, view the archive [here](https://www.freelists.org/archive/comixed/).

**Comment by Bugmaniac71** [comment]
Hi Daryl, thanks for teh prompt response ;->
1. Say anywhare where I need to select 1 comic or more. Cover view is well and good but slow. Take Comic Rack for instance - default for me is a list view where I can select any individual or multi select. In fact thinking about it the only place using ComicRack I used a cover was during the scrape to check that the issue was correct, Everything else was a list view.

3. Thanks - it may be just my system but both Comixed and ComicRack have issues with 'some' CBR releases. TBH it's pushed me to finding a script that converts to CBX BEFORE importing - works much, much smoother for me (and also, said script optionally converts to WEBP as well so thats going to save me about a TB of diskspace once I've converted all y stuff over!)

4. Probably just me but having multiple info for 1 comic on separate tabs is a pain (to me!)

5. Especially as I'm just getting used to the layouts - made a couple of mistakes when selecting the correctly scraped files.

6. Opened issue for this one.
Yes just found the archived mailing list and can access that so will have a trawl.
Other than that great app - glad to see someone is taking the time to push our Comic hobby further (and also glad its cross-platform too - added bonus!)
Regards
B
